erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -727,11 +727,6 @@ def get_columns(self):
 			self.add_column(_('{0} Name').format(self.party_type),
 				fieldname = scrub(self.party_type) + '_name', fieldtype='Data')
 
-		if self.party_type == 'Customer':
-			self.add_column(_("Customer Contact"), fieldname='customer_primary_contact',
-				fieldtype='Link', options='Contact')
-
-		self.add_column(label=_('Voucher Type'), fieldname='voucher_type', fieldtype='Data')
 		self.add_column(label=_('Voucher No'), fieldname='voucher_no', fieldtype='Dynamic Link',
 			options='voucher_type', width=180)
```

Comment:
@rvpasari you are using the Dynamic Link field here which refers to voucher_type which is removed from code. Make sure it has correct Options. it might break linking.

Reply:
Tested this, it seems to be working fine